src/iris.ts

```diff
@@ -0,0 +1,61 @@
+import cvSetsK5 from '../data/cvK5.json';
+import cvSetsK7 from '../data/cvK7.json';
+import raw from '../data/iris.json';
+
+import getNumericalClasses from './utilities/getNumericalClasses';
+
+const dataset = raw as [number, number, number, number, string][];
+
+export function getDataset(): (number | string)[][] {
+  return dataset.slice();
+}
+
+export function getNumbers(): number[][] {
+  return dataset.map((sample) => sample.slice(0, 4) as number[]);
+}
+
+export function getClasses(): string[] {
+  return dataset.map((sample) => sample[4]);
+}
+
+export function getClassesAsNumber(): number[] {
+  const classes = dataset.map((sample) => sample[4]);
+  return getNumericalClasses(classes);
+}
+
+export function getDistinctClasses(): [string, string, string] {
```

Comment:
Please don't hardcode the answer and just recalculate it using your `Set` approach.